src/components/AddressInputPanel/index.tsx

```diff
@@ -108,6 +108,7 @@ export default function AddressInputPanel({
         <InputContainer>
           <AutoColumn gap="md">
             <RowBetween>
+              {/*Mod: theme.text2 => theme.text1*/}
```

Comment:
To create a Mod you have to duplicate the file instead of modifying the original

You would need to create these two according to the [guide](https://docs.google.com/document/d/1dcAqCCuKBIzOiqw3ppXHXRX86s9gvj3BvXuVqAgookg/edit):
- `src/custom/components/AddressInputPanel/AddressInputPanel.tsx`
- `src/custom/components/AddressInputPanel/index.tsx`

Reply:
Thanks!
Already done it here: https://github.com/cowprotocol/cowswap/pull/1040/commits/2d16fe9ae791b61d00242df5da698748291c2075